<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vira" name="Cancle Mail" tests="1" failures="1" timestamp="5 Nov 2015 18:22:56 GMT" time="19.351" errors="0">
  <testcase name="failedMethod" time="0.0" classname="com.gmail.verifications.Index">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.gmail.verifications.Index.failedMethod(Index.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- failedMethod -->
</testsuite> <!-- Cancle Mail -->
